assets/js/calypsoify.js

```diff
@@ -11,10 +11,15 @@
         }
     } );
 
+// TODO Remove the notice handlers after https://github.com/Automattic/jetpack/pull/13126 has been released.
     /**
      * Append icons to notices
      */
     $( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
+	if ( $( this ).children( '.wc-calypso-bridge-notice-content' ).length ) {
```

Comment:
I'm seeing some indentation issues here.  (I think this whole file might need to be reindented as it appears to be using spaces).

Reply:
The spacing in the whole file updated in https://github.com/Automattic/wc-calypso-bridge/pull/491/commits/7b1b7a43c828edb61edaed07519cb6dac44b5b3a